apps/aecontract/src/aect_evm.erl

```diff
@@ -83,11 +85,13 @@ execute_call(#{ code := Code
               , currentTimestamp := TS
               , chainState := ChainState
               , chainAPI := ChainAPI
+              , type_info := TypeInfo
               , vm_version := VmVersion
               }, Trace) ->
     %% TODO: Handle Contract In State.
     Spec =
         #{ exec => #{ code => Code
+                    , type_info => TypeInfo
```

Comment:
I'm not sure we need the type info in the VM state. Once we get to running VM code the types should already be checked, no?

Reply:
Hmm, that is probably correct. The type info should be checked when the byte code of the next contract is fetched, so we don't need it here. I can amend this.